src/operator/nn/mkldnn/mkldnn_fully_connected.cc

```diff
@@ -42,9 +42,10 @@ mkldnn::inner_product_forward::primitive_desc GetFCFwdImpl(const MKLDNNFCFullPar
                                                            const NDArray* bias,
                                                            const mkldnn::memory::desc& out_md) {
   auto data_md   = GetMemDesc(data);
-  auto weight_md = full_param.mkldnn_param.quantized ? GetFCWeightDesc(weight, mshadow::kInt8)
-                                                     : GetFCWeightDesc(weight);
-  auto engine    = CpuEngine::Get()->get_engine();
+  auto weight_md = full_param.mkldnn_param.quantized
+                       ? GetFCWeightDesc(weight, data.shape()[0], mshadow::kInt8)
+                       : GetFCWeightDesc(weight, data.shape()[0]);
+  auto engine = CpuEngine::Get()->get_engine();
```

Comment:
This line could stay aligned on "="

Reply:
This is how clang-formatter works, should I ignore it and align manually?

Reply:
Please check if you have code formatted according to https://github.com/apache/incubator-mxnet/pull/20433/files

Reply:
It only align variables that are one below another without additional lines between. I moved engine to the top of the function and it's aligned now

Reply:
Thanks